examples/bert/README.md

```diff
@@ -178,3 +179,24 @@ tensorboard --logdir runs/
 ```
 
 ![Visualizing loss/accuarcy on Tensorboard](tbx.png)
+
+## Hyperparameter tuning with Executor
+
+To run this example, please install `hyperopt` by issuing the following command
+
+```commandline
+pip install hyperopt
+```
+
+`bert_with_tpe.py` shows an example of how to tune hyperparameters with Executor using `hyperopt`. 
+To run this example, run the following command
+
+```commandline
+python bert_with_tpe.py
```

Comment:
probably name it `bert_with_hypertuning_main.py` to show this is a main entry

Reply:
replace `tpe` with `hypertuning` in the file names to make easier understanding

Reply:
Fixed in e175fa6